.buildkite/scripts/common.sh

```diff
@@ -21,7 +41,8 @@ add_bin_path() {
 with_go() {
     echo "Setting up the Go environment..."
     create_workspace
-    retry 5 curl -sL -o ${WORKSPACE}/gvm "https://github.com/andrewkroh/gvm/releases/download/${SETUP_GVM_VERSION}/gvm-linux-amd64"
+    check_platform_architeture
+    retry 5 curl -sL -o ${WORKSPACE}/gvm "https://github.com/andrewkroh/gvm/releases/download/${SETUP_GVM_VERSION}/gvm-${platform_type,,}-${arch_type}"
```

Comment:
```suggestion
    retry 5 curl -sL -o ${WORKSPACE}/gvm "https://github.com/andrewkroh/gvm/releases/download/${SETUP_GVM_VERSION}/gvm-${platform_type}-${arch_type}"
```

Reply:
No, it isn't a typo, this helps to transform output from `Linux` to `linux`, because the output of the command `uname` is `Linux` or `Darwin`